test/Mono.Linker.Tests.Cases/Reflection/ActivatorCreateInstance.cs

```diff
@@ -1,11 +1,13 @@
 using System;
 using System.Reflection;
+using System.Runtime.CompilerServices;
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
 using Mono.Linker.Tests.Cases.Expectations.Metadata;
 
 namespace Mono.Linker.Tests.Cases.Reflection
 {
 	[SetupCSharpCompilerToUse ("csc")]
+	[KeptMember(".ctor()")]
```

Comment:
Why is this constructor kept now?

Reply:
Because the test instantiates the test class (see below in `Main`). This was the simplest way as there's a rather tricky behavior difference in Cecil calling static versus instance methods, so I wanted to make sure that the logic works correctly in both cases.